plugins/view-resources/src/components/Table.svelte

```diff
@@ -203,6 +203,7 @@
   }
 
   $: checkedSet = new Set<Ref<Doc>>(checked.map((it) => it._id))
+  $: allItemsSelected = objects?.length === checkedSet.size && objects?.length > 0
```

Comment:
Does object contain all objects or only current "page"? Maybe we should compare with `gtotal` number instead?